src/index.html

```diff
@@ -22,6 +22,389 @@
+            <div class="mark__stars">
+              <span class="star"></span>
+              <span class="star"></span>
+              <span class="star"></span>
+              <span class="star"></span>
+              <span class="star"></span>
+            </div>
+            <div class="mark__reviews">Reviews: 5</div>
+          </div>
+          <div class="price">
+            <div class="price__word">Price:</div>
+            <div class="price__numeric">$2,199</div>
+          </div>
+          <a
+            href="#"
+            data-qa="hover"
+            class="bottom"
+          >
+            BUY
+          </a>
+        </div>
+      </div>
+      <div class="product-cards">
+        <div
+          class="product-card"
+          data-qa="card"
+        >
+          <img
+            src="./images/imac.jpeg"
+            alt="imac"
+            class="product-card__image"
+          />
+          <h2 class="product-card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <p class="product-card__description">Product code: 195434</p>
+          <div class="mark">
+            <div class="mark__stars">
+              <span class="star"></span>
+              <span class="star"></span>
+              <span class="star"></span>
+              <span class="star"></span>
+              <span class="star"></span>
+            </div>
+            <div class="mark__reviews">Reviews: 5</div>
+          </div>
+          <div class="price">
+            <div class="price__word">Price:</div>
+            <div class="price__numeric">$2,199</div>
+          </div>
+          <a
+            href="#"
+            data-qa="hover"
+            class="bottom"
+          >
+            BUY
+          </a>
+        </div>
+      </div>
+      <div class="product-cards">
+        <div
+          class="product-card"
+          data-qa="card"
+        >
+          <img
+            src="./images/imac.jpeg"
+            alt="imac"
+            class="product-card__image"
+          />
+          <h2 class="product-card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h2>
+          <p class="product-card__description">Product code: 195434</p>
+          <div class="mark">
+            <div class="mark__stars">
+              <span class="star"></span>
+              <span class="star"></span>
+              <span class="star"></span>
+              <span class="star"></span>
+              <span class="star"></span>
+            </div>
+            <div class="mark__reviews">Reviews: 5</div>
+          </div>
+          <div class="price">
+            <div class="price__word">Price:</div>
+            <div class="price__numeric">$2,199</div>
+          </div>
+          <a
+            href="#"
+            data-qa="hover"
+            class="bottom"
+          >
+            BUY
+          </a>
+        </div>
+      </div>
+      <div class="product-cards">
```

Comment:
Add the blank line between the cards
```suggestion
      </div>
      
      <div class="product-cards">
```